### DISCUSSION TITLE
How to inject a custom dropdown in sidebar header?

### DISCUSSION THREAD

**Discussion by grelas** [storybookjs/storybook#17023] [Help]
Looking at the addons page here https://github.com/storybookjs/storybook/tree/next/addons, the screenshot shows a dropdown (presumably to change versions). Which addon is this? If this is not an addon, how can I achieve this? I'd like to be able to inject a custom dropdown with a list of links.

![image](https://user-images.githubusercontent.com/993698/146249810-ae6e1f45-8dc9-4baf-a25a-96aac11c1855.png)

**Comment by jonniebigodes** [comment]
@grelas that is not actually an addon, but in fact Storybook's [composition](https://storybook.js.org/docs/react/sharing/storybook-composition) being shown. To the best of my knowledge, there's no addon that does what you intend to achieve.
